src/command.c

```diff
@@ -277,12 +277,21 @@ static bool cmd_target_power(target *t, int argc, const char **argv)
 #endif
 
 #ifdef PLATFORM_HAS_TRACESWO
-static bool cmd_traceswo(void)
+static bool cmd_traceswo(target *t, int argc, const char **argv)
 {
-	extern char serial_no[9];
-	traceswo_init();
-	gdb_outf("%s:%02X:%02X\n", serial_no, 5, 0x85);
-	return true;
+  extern char serial_no[9];
+  uint32_t speed=0;
+  (void)t;
+
+  if (argc>1)
+    speed=atoi(argv[1]);
+
+  traceswo_init(speed);
+  if (!speed)
+    {
+      gdb_outf("%s:%02X:%02X\n", serial_no, 5, 0x85);
```

Comment:
This information was output so that a GDB python extension could identify the USB endpoint for trace.  I don't see why it should be suppressed if speed is specified.